src/tests/fuzzing/fuzz_card.c

```diff
@@ -86,8 +86,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 	sc_list_files(card, files, sizeof(files));
 
 	/* Get challenge */
-	if ((rnd = malloc(len)))
-	sc_get_challenge(card, rnd, len);
+	rnd = malloc(len);
+	if (rnd == NULL)
+		goto err;
+	if (sc_get_challenge(card, rnd, len) != SC_SUCCESS)
```

Comment:
Is it necessary to check whether `sc_get_challenge()` fails here? The return values are not tested for the other functions since they are independent.

Reply:
coverity complains about not checking return values. I think just logging something in the condition might be enough to make coverity happy

Reply:
Ok, looks good.